py/desitarget/mock/io.py

```diff
@@ -476,7 +470,7 @@ def read_mock_bgs_mxxl_brighttime(root_mock_dir='',mock_prefix='',brickname_list
     full_data   = dict()
     n_per_file  = list()
     if len(target_list) > 0:
-        for k in target_list[0].keys():
+        for k in list(target_list[0]):
```

Comment:
minor: if `target_list[0]` is a dictionary, then `for k in target_list[0]:` iterates over the keys without requiring `list(...)` or `.keys()` and works in both python 2.7 and 3.5 without 2to3 complaining about it.